Application/EdFi.Ods.Standard/Artifacts/Schemas/SchemaAnnotation.xsd

```diff
@@ -1,3 +1,10 @@
+<!--
+   SPDX-License-Identifier: Apache-2.0
+   Licensed to the Ed-Fi Alliance under one or more agreements.
+   The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+   See the LICENSE and NOTICES files in the project root for more information.
+-->
```

Comment:
license headers don't need to be on .xml files. for reference there are file types that are out of scope that are listed here: https://techdocs.ed-fi.org/display/EDFIODS/License+and+Copyright+Notifications

Reply:
Done.  I think I removed all of the headers from xml and xsd.